internal/cmd/base/delete.go

```diff
@@ -85,8 +85,13 @@ func (dc *DeleteCmd) Run(s state.State, cmd *cobra.Command, args []string) error
 			}
 		}
 
+		for _, result := range results {
+			if result.Error != nil {
+				errs = append(errs, result.Error)
+			}
+		}
```

Comment:
What do you think about combining this with the existing loop of `results` in Line 95/100